tests/test_library/demo/github_rep.yaml

```diff
@@ -0,0 +1 @@
+- type: github-pr
```

Comment:
Hmmm, I'm not sure what's the purpose of this file. It looks like it was added by accident?

Reply:
It was used for demonstration in [https://github.com/velezd/gh-actions-test/pull/1](https://github.com/velezd/gh-actions-test/pull/1). I can remove it now, if you want.

Reply:
How was this file used - I mean where in the code this file could be used? I don't see any reference in the code and I don't see a way how this could be used by the tplib library either.

Reply:
`-o reportSenders.additional_reporting=library://github_rep.yaml`

Reply:
Ok, I get it now, I was little bit afraid it would be something like this :) Yeah, I'd say let's remove this file from the demo library as the purpose of the file is not very clear and it doesn't do anything without this additional command line option.